core/common/src/main/java/alluxio/wire/FileInfo.java

```diff
@@ -251,17 +253,31 @@ public String getUfsFingerprint() {
   }
 
   /**
-   * @return the ACL entries for this file
+   * @return the ACL object for this file
+   */
+  public AccessControlList getAcl() {
+    return mAcl;
+  }
+
+  /**
+   * @return the default ACL object for this file
    */
-  public List<String> getAclEntries() {
-    return mAclEntries;
+  public DefaultAccessControlList getDefaultAcl() {
+    return mDefaultAcl;
   }
 
   /**
-   * @return the default ACL entries for this file
+   * @return the ACL as string entries for this file
    */
-  public List<String> getDefaultAclEntries() {
-    return mDefaultAclEntries;
+  public List<String> convertAclToStringEntries() {
+    return (mAcl == null) ? new ArrayList<>() : mAcl.toStringEntries();
+  }
+
+  /**
+   * @return the default ACL as string entries for this file
+   */
+  public List<String> convertDefaultAclToStringEntries() {
```

Comment:
NIT: `getDefaultAclStringEntries`

Reply:
I purposely avoided using `getX` because it will be used by the json serializer and deserializer if it is named `getX`.

Reply:
We should add a comment to not use `get` for this name.